### ISSUE TITLE
[Todoist] ...video of problem with connectino to raycast

### ISSUE THREAD

**Issue by egoldy** [raycast/extensions#14585]
### Extension

https://www.raycast.com/doist/todoist

### Raycast Version

latest 

### macOS Version

15.1

### Description

Trying to connect to raycast to get a quick atask added.  Gives me an error

### Steps To Reproduce

https://share.zight.com/5zuoAQGd

See this video

### Current Behaviour

Watch this video  https://share.zight.com/5zuoAQGd

Always comes up  when try and connect it.

### Expected Behaviour

Should be able to create a task from Raycast.

**Comment by thomaslombart** [comment]
Can you copy the error log you're getting? You can do that by opening the action panel (⌘ + K) when you encounter the error screen.

**Comment by egoldy** [comment]
Here is the error log:

FetchError: request to https://todoist.oauth-proxy.raycast.com/token failed, reason: unable to verify the first certificate

    at withAccessToken(WC)
    at Ht
_r:index.js:8:2593
    at ray-navigation-stack

**Comment by thomaslombart** [comment]
Are you using some kind of VPN/firewall/proxy?

**Comment by egoldy** [comment]
Aha….

Not a VPN, firewall, proxy (I am running Little Snitch which I disabled and didn’t’ do anything).

HOWEVER, I do have AdBlocker running on the mac and THAT was causing the problem.   It was blocking the forwarding to the other page.

So, bottom line, I have it working, thank you.

As an FYI - I did try (I think I had tried before) to copy the link (there’s an option) and that didn’t seem to provide a link I could copy and paste into Safari.

But, I have it working!

Thanks,

Eric
